.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
```

Comment:
These `-`s on their own line might be valid YAML but they look weeeird

Reply:
Agreed